tests/smartcard_server_role/test_smartcard_server_role.yml

```diff
@@ -0,0 +1,185 @@
+---
+- name: Test group
```

Comment:
Shouldn't it be "Test smartcard server role" ?

Reply:
Will change to "Test smartcard_server role"